trigger/categories/lib.php

```diff
@@ -118,17 +118,20 @@ public function instance_settings() {
     public function extend_add_instance_form_definition($mform) {
         global $DB;
         $categories = $DB->get_records('course_categories');
-        $categorynames = array();
-        foreach ($categories as $category) {
-            $categorynames[$category->id] = $category->name;
-        }
+        //$categorynames = array();
+        //foreach ($categories as $category) {
+        //    $categorynames[$category->id] = $category->name;
+        //}
         $options = array(
             'multiple' => true,
             'noselectionstring' => get_string('categories_noselection', 'lifecycletrigger_categories'),
         );
+        //Cangelog: Added core course lookup functionality, with flat hierachical representation
```

Comment:
Nur was winziges, aber ich würde keinen Changelog als Kommentar in den Code schreiben, sondern eher einfach beschreiben, was die neue Zeile macht ('Get course categories with flat hierachical representation.', oder so)

Der [CI-Workflow](https://github.com/learnweb/moodle-tool_lifecycle/runs/4459602709?check_suite_focus=true#step:13:37) schlägt gerade auch wegen dem [Moodle Codestyle bei Kommentaren](https://docs.moodle.org/dev/Coding_style#Inline_comments) fehl